### PR TITLE
Update presence-admins.md

### PR DESCRIPTION
**Pull request by RuiPereiraTabares**
updating the article as we dont have a mention of https://docs.microsoft.com/en-us/powershell/module/skype/set-csprivacyconfiguration?view=skype-ps if the value is in true the presence will not be published.

### CODE REVIEW

**Comment by ndambrosio** [comment]
Pretty sure this has no influence on Microsoft Teams, this is used for Skype for Business only.

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks

File: Teams/presence-admins.md [hunk 1]
```diff
@@ -25,7 +25,8 @@ Presence is part of a user's profile in Microsoft Teams (and throughout Microsof
 
  > [!Note]
  > For details about Teams user profiles on different platforms, see [Teams features by platform](https://support.microsoft.com/office/teams-features-by-platform-debe7ff4-7db4-4138-b7d0-fcc276f392d3).
-
+ > [!Note]
```
Comment (JohanFreelancer9) [inline Teams/presence-admins.md:28]:
```suggestion
```

Comment (JohanFreelancer9) [inline Teams/presence-admins.md:28]:
```suggestion

 > [!NOTE]
```

File: Teams/presence-admins.md [hunk 2]
```diff
@@ -25,7 +25,8 @@ Presence is part of a user's profile in Microsoft Teams (and throughout Microsof
...
+ > Teams respects the privacy configuration so if you have enabled the privacy mode your presence will not be published for external users
```
Comment (JohanFreelancer9) [inline Teams/presence-admins.md:29]:
```suggestion
 > Teams respects the privacy configuration so if you have enabled the privacy mode, your presence will not be visible to external users.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by scanum** [comment]
@msdmaguire PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by JohanFreelancer9** [comment]
@RuiPereiraTabares - please commit my suggested changes. Many thanks.

**Comment by RuiPereiraTabares** [comment]
@JohanFreelancer9 done

**Comment by JohanFreelancer9** [comment]
@RuiPereiraTabares - please also click on the commit button for the other two suggested changes. Thanks.

**Comment by RuiPereiraTabares** [comment]
@JohanFreelancer9 done with both

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by RuiPereiraTabares** [comment]
@msdmaguire can you check and approve the pr?

**Review by cazawideh — APPROVED** [review]

**Pull request merged**